### PR TITLE
CIV-10379 additional logs added

### PR DESCRIPTION
**Pull request by kalachandrasekar1**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-10379

### Change description ###
Additional logs added for testing

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/civil/service/stitching/CivilDocumentStitchingService.java [hunk 1]
```diff
@@ -38,6 +40,9 @@ public class CivilDocumentStitchingService implements DocumentStitcher {
     public CaseDocument bundle(List<DocumentMetaData> documents, String authorisation, String bundleTitle, String bundleFilename, CaseData caseData) {
         CaseDetails payload = createBundlePayload(documents, bundleTitle, bundleFilename, caseData);
         log.info("Calling stitching api end point for {}", caseData.getLegacyCaseReference());
+        convertToJson(payload);
```
Comment (neeta-hmcts) [inline src/main/java/uk/gov/hmcts/reform/civil/service/stitching/CivilDocumentStitchingService.java]: remove logs

**Review by neeta-hmcts — DISMISSED** [review]

**Review by laperla11 — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/service/stitching/CivilDocumentStitchingService.java [hunk 2]
```diff
@@ -51,23 +53,31 @@ public CaseDocument bundle(List<DocumentMetaData> documents, String authorisatio
...
         String documentUrl = document.getDocumentUrl();
         String documentBinaryUrl = document.getDocumentBinaryUrl();
+
+        LocalDateTime responseDate = null;
+        if (Objects.nonNull(caseData.getRespondent1ResponseDate())) {
+            responseDate = caseData.getRespondent1ResponseDate();
+        } else if (Objects.nonNull(caseData.getRespondent2ResponseDate())) {
+            responseDate = caseData.getRespondent2ResponseDate();
+        }
+
```
Comment (sankhajuria) [inline src/main/java/uk/gov/hmcts/reform/civil/service/stitching/CivilDocumentStitchingService.java:75]: whats the use of this code ?

**Review by sankhajuria — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/service/stitching/CivilDocumentStitchingService.java [hunk 3]
```diff
@@ -51,23 +53,31 @@ public CaseDocument bundle(List<DocumentMetaData> documents, String authorisatio
...
         return CaseDocument.builder()
             .documentLink(Document.builder().documentUrl(documentUrl).documentBinaryUrl(documentBinaryUrl).documentFileName(document.getDocumentFileName()).build())
             .documentName("Stitched document")
             .documentType(SEALED_CLAIM)
-            .createdDatetime(LocalDateTime.now())
+            .createdDatetime(caseData.getCreatedResponseDate(responseDate))
```
Comment (TatsyMaslova) [inline src/main/java/uk/gov/hmcts/reform/civil/service/stitching/CivilDocumentStitchingService.java:80]: its the caseData that has to return the date without us having to retrieve the parameter. the knowledge is already in casedata

File: src/main/java/uk/gov/hmcts/reform/civil/model/CaseData.java [hunk 4]
```diff
@@ -1000,4 +1000,10 @@ && getApplicant1ResponseDeadline().isBefore(LocalDateTime.now())
     public String getApplicant1Email() {
         return getApplicant1().getPartyEmail() != null ? getApplicant1().getPartyEmail() : getClaimantUserDetails().getEmail();
     }
+
+    @JsonIgnore
+    public LocalDateTime getCreatedResponseDate(LocalDateTime responseDate) {
+        return LocalDateTime.now().isBefore(responseDate) ? responseDate : LocalDateTime.now();
```
[the remark this answers is not in the document]
Comment (kalachandrasekar1) [reply src/main/java/uk/gov/hmcts/reform/civil/model/CaseData.java:1012]: It's moved to caseData

File: src/main/java/uk/gov/hmcts/reform/civil/service/stitching/CivilDocumentStitchingService.java (hunk 2 above)
Comment (kalachandrasekar1) [reply src/main/java/uk/gov/hmcts/reform/civil/service/stitching/CivilDocumentStitchingService.java]: Compare with LocalDateTime.now() to get the respondent date, this change added.Now moved this change to caseData

File: src/main/java/uk/gov/hmcts/reform/civil/service/stitching/CivilDocumentStitchingService.java (hunk 3 above)
Comment (kalachandrasekar1) [reply src/main/java/uk/gov/hmcts/reform/civil/service/stitching/CivilDocumentStitchingService.java]: done

**Review by TatsyMaslova — APPROVED** [review]

**Review by sankhajuria — APPROVED** [review]

**Review by neeta-hmcts — APPROVED** [review]

**Pull request merged**